manager/controllers/app/fybrikapplication_controller_test.go

```diff
@@ -63,6 +69,10 @@ var _ = Describe("FybrikApplication Controller", func() {
 			}
 		})
 		It("Test end-to-end for FybrikApplication", func() {
+			connector := os.Getenv("USE_WKC_CONTROLLER")
+			if len(connector) > 0 && connector == "true" {
+				return
+			}
```

Comment:
Why this condition was added?

Reply:
We often deploy with the Watson Studio Connector.  This test does not work when it is configured.

Reply:
Currently, we're adding tests for that connector in a separate repo, but if you feel that test belongs in this repo, let me know.

Reply:
Would it make sense to replace with connector != "true" which will cover unset variable?

Reply:
done